### PR TITLE
[ANCHOR-868] Fix broken docker-compose file

### PR DESCRIPTION
**Pull request by lijamie98**
### Description

- Make KT_REFERENCE_SERVER_CONFIG optional and default to the `reference_config.yaml` file.
- Add `v2_stable` and `latest` profiles to docker_compose.yaml
- Fix Kafka platform problem for MacOS
- Modify the `README.md` file

### Context

- Fix broken `docker_compose.yaml file`
- Improve developer experience

### Testing

- `./gradlew test`

### Documentation

N/A

### Known limitations

This will break `typescript-wallet-sdk`. The PR to fix `typescript-wallet-sdk` is https://github.com/stellar/typescript-wallet-sdk/pull/181

### CODE REVIEW

File: .run/Test Profile_ auth-jwt-custody.run.xml [hunk 1]
```diff
@@ -1,12 +1,11 @@
 <component name="ProjectRunConfigurationManager">
   <configuration default="false" name="Test Profile: auth-jwt-custody" type="JetRunConfigurationType" folderName="Run Test Profile">
     <envs>
-      <env name="app.enableTest" value="true" />
       <env name="TEST_PROFILE_NAME" value="auth-jwt-custody" />
-      <env name="KT_REFERENCE_SERVER_CONFIG" value="service-runner/src/main/resources/config/reference-config.yaml" />
     </envs>
     <option name="MAIN_CLASS_NAME" value="org.stellar.anchor.platform.run_profiles.RunTestProfile" />
     <module name="java-stellar-anchor-sdk.service-runner.main" />
+    <option name="REDIRECT_INPUT" value="true" />
```
Comment (philipliu) [inline .run/Test Profile_ auth-jwt-custody.run.xml:8]: Why is this required for this profile?

File: docs/README.md [hunk 2]
```diff
@@ -51,12 +51,18 @@ contribute to this project.
...
 ## Quickstart
 
-Anchor Platform can be run locally using Docker Compose. This will start an instance of the Anchor Platform and the
-Kotlin reference server.
+Anchor Platform can be run locally using Docker Compose. 
 
+### For version 2.x.x stable release
 ```shell
-docker build --build-arg BASE_IMAGE=gradle:7.6.4-jdk17 -t anchor-platform:local ./
-docker compose -f service-runner/src/main/resources/docker-compose.yaml up -d
```
Comment (philipliu) [inline docs/README.md:59]: Should we provide a way for developers to start the stack against a locally built image?

File: service-runner/src/main/resources/common/docker-compose.yaml [hunk 3]
```diff
@@ -1,4 +1,16 @@
...
+    environment:
+      - KT_REFERENCE_SERVER_CONFIG=/config/reference-config.yaml
+    command: -t
+    volumes:
+      - ../config:/config
+    ports:
+      - "8080:8080" # SEP server
+      - "8085:8085" # Platform server
+      - "8091:8091" # Reference server
+      - "8092:8092" # Reference server
```
Comment (philipliu) [inline service-runner/src/main/resources/common/docker-compose.yaml:12]: nit: can we distinguish which reference server?

File: service-runner/src/main/kotlin/org/stellar/anchor/platform/run_profiles/RunKotlinReferenceServer.kt [hunk 4]
```diff
@@ -4,9 +4,20 @@ package org.stellar.anchor.platform.run_profiles
 
 import kotlinx.coroutines.runBlocking
 import org.stellar.anchor.platform.*
+import org.stellar.reference.di.ConfigContainer.Companion.KT_REFERENCE_SERVER_CONFIG
 
 fun main() = runBlocking {
-  testProfileExecutor = TestProfileExecutor(TestConfig())
+  testProfileExecutor =
```
Comment (philipliu) [inline service-runner/src/main/kotlin/org/stellar/anchor/platform/run_profiles/RunKotlinReferenceServer.kt:10]: Why is this also required in `RunTestProfile.kt`?

File: .run/Reference Server_ default.run.xml [hunk 5]
```diff
@@ -1,9 +1,7 @@
 <component name="ProjectRunConfigurationManager">
   <configuration default="false" name="Reference Server: default" type="JetRunConfigurationType" folderName="Run Single Server">
     <envs>
-      <env name="app.enableTest" value="true" />
```
Comment (philipliu) [inline .run/Reference Server_ default.run.xml:4]: Why is it not required anymore?

File: .run/Test Profile_ auth-jwt-custody.run.xml (hunk 1 above)
Comment (lijamie98) [reply .run/Test Profile_ auth-jwt-custody.run.xml:8]: Not sure. This was added by IntelliJ. 

I have removed it manually.

File: service-runner/src/main/kotlin/org/stellar/anchor/platform/run_profiles/RunKotlinReferenceServer.kt (hunk 4 above)
Comment (lijamie98) [reply service-runner/src/main/kotlin/org/stellar/anchor/platform/run_profiles/RunKotlinReferenceServer.kt:10]: This is to default KT_REFERENCE_SERVER_CONFIG to "service-runner/src/main/resources/config/reference-config.yaml" to reduce the verboseness in running the test profiles.

File: .run/Reference Server_ default.run.xml (hunk 5 above)
Comment (lijamie98) [reply .run/Reference Server_ default.run.xml:4]: The `enableTest` is already defined in [reference-config.yaml ](https://github.com/lijamie98/java-stellar-anchor-sdk/blob/feature/anchor-868-servicerunner-run-test-profile/service-runner/src/main/resources/config/reference-config.yaml#L17)

File: docs/README.md [hunk 6]
```diff
@@ -51,12 +51,31 @@ contribute to this project.
...
+```shell
+# In the root directory of the project
+docker compose --profile latest up -d
+```
+
+### For locally built image
+To build the Anchor Platform image locally, run the following command in the root directory of the project.
+```shell
+  # In the root directory of the project
+docker build -t stellar/anchor-platform .
```
Comment (philipliu) [inline docs/README.md:72]: Do we need to add the `:local` tag here?

**Review by philipliu — APPROVED** [review]
Minor comment
